### PR TITLE
Remove empty if statements

### PR DESCRIPTION
**Pull request by xavier-figueroa**
Remove empty if statements
majorAxis is equals to SplitPanel.MAJOR_AXIS_VERTICAL always

### CODE REVIEW

File: src/com/lightdev/app/shtm/SplitPanel.java [hunk 1]
```diff
@@ -123,23 +112,18 @@ private void buildLayout() {
     public void removeAllOuterPanels() {
         // Warning: it does not really remove the outer panels per se.
         JComponent p;
-        if (majorAxis == MAJOR_AXIS_VERTICAL) {
```
Comment (dpolivaev) [inline src/com/lightdev/app/shtm/SplitPanel.java]: Why have you removed this if statement?

File: src/com/lightdev/app/shtm/SplitPanel.java [hunk 2]
```diff
@@ -262,50 +246,33 @@ public void adjustDividerSizes() {
      * @param location  the location to add to (SplitPanel.NORTH, SOUTH, etc.)
      */
     public void addComponent(final JComponent component, final int location) {
-        if (majorAxis == SplitPanel.MAJOR_AXIS_VERTICAL) {
```
Comment (dpolivaev) [inline src/com/lightdev/app/shtm/SplitPanel.java]: Same question

File: src/com/lightdev/app/shtm/SplitPanel.java (hunk 1 above)
Comment (xavier-figueroa) [reply src/com/lightdev/app/shtm/SplitPanel.java]: at initialization
```
public static final int MAJOR_AXIS_VERTICAL = 2;
...
private final int majorAxis = SplitPanel.MAJOR_AXIS_VERTICAL;
```
both variables will always have the same value, only the if block will be executed always

File: src/com/lightdev/app/shtm/SplitPanel.java (hunk 2 above)
Comment (xavier-figueroa) [reply src/com/lightdev/app/shtm/SplitPanel.java]: same response :)

File: src/com/lightdev/app/shtm/SplitPanel.java (hunk 1 above)
Comment (dpolivaev) [reply src/com/lightdev/app/shtm/SplitPanel.java]: Do you think we should also remove MAJOR_AXIS_HORIZONTAL "NOT SUPPORTED" ?

Comment (xavier-figueroa) [reply src/com/lightdev/app/shtm/SplitPanel.java]: yeah, I think we can remove
MAJOR_AXIS_HORIZONTAL
MAJOR_AXIS_VERTICAL
majorAxis
none of them are being used. I can do it later today

**Pull request closed without merging**